psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -1562,6 +1563,46 @@ std::vector<SharedMatrix> MintsHelper::electric_field(const std::vector<double>
     return field;
 }
 
+SharedMatrix MintsHelper::induction_operator(SharedMatrix coords, SharedMatrix moments) {
+    SharedMatrix mat = std::make_shared<Matrix>("Induction operator", basisset_->nao(), basisset_->nao());
```

Comment:
This call automatically zeroes out the matrix so the `std::fill_n` above would be redundant in this case.